src/conversions.rs

```diff
@@ -553,7 +576,7 @@ impl<C: Clone, T: FromJSValConvertible<Config=C>> FromJSValConvertible for Vec<T
         let iterator = &mut *iterator.root;
 
         if !iterator.init(value, ForOfIterator_NonIterableBehavior::ThrowOnNonIterable) {
-            return Err(())
+            return Ok(ConversionResult::Failure(Cow::Owned("Not iterable".to_owned())))
```

Comment:
This doesn't agree with the `ThrowOnNonIterable` argument passed above. Please keep returning `Err(())`